### PR TITLE
Fixes README instructions for running tests

### PR DESCRIPTION
**Pull request by filipeximenes**
`poetry run pytest` on a fresh install would not work and there were no instructions pointing that you need to set `OPENAI_API_KEY` on in `.env.tests` for the tests to pass. This PRs adds instructions to create a `.env.tests` file.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -97,19 +104,21 @@ poetry run mkdocs serve
 ## Release
 
 !!! info
-    The backend and the frontend are versioned together, that is, they should have the same version number.
+The backend and the frontend are versioned together, that is, they should have the same version number.
```
Comment (pamella) [inline CONTRIBUTING.md:107]: Please undo this change because it breaks the formatting.

**Review by pamella — CHANGES_REQUESTED** [review]

**Review by fjsj — CHANGES_REQUESTED** [review]
Overall LGTM, but please check Pam's comment.

**Review by pamella — APPROVED** [review]
LGTM
